help/md/dnCDBDP.md

```diff
@@ -1,9 +1,82 @@
 ## name
 dnCDBDP
 ## title
+Character Dependent Birth Death Process
 ## description
+This function simulates a tree under a character-dependent birth-death process.
 ## details
+This function is a flexible simulator that can be used for several phylogenetic models. Specifically,
+dnCDBDP allows for character-dependent birrth-death processes meaning that the speciation/extinction rates
+depend on the character state of the taxon itself.
+ Examples of such models are outlined below:
+
+Multiple State-dependent Speciation Extinction (MuSSE)
+This model uses a state-dependent birth-death process to simulate a tree with only anagentic state changes.
+Using dnCDBDP to implement MuSSE, a vector of speciation rates for each state can be passed to the lambda argument.
+Due to being a vector of rates, dnCDBDP will anly allow anagenetic state changes (along branches), with the
```

Comment:
Typos: "anagentic" should read "anagenetic" (line 14), "anly" should read "only" (line 16). Also, the last sentence of the MuSSE description is somewhat awkward, and implies that `dnCDBDP` is a vector of rates – it's not; it is a distribution.